man/dockerd.8.md

```diff
@@ -78,6 +80,12 @@ format.
 **--api-cors-header**=""
   Set CORS headers in the remote API. Default is cors disabled. Give urls like "http://foo, http://bar, ...". Give "*" to allow all.
 
+**--authn**=*true*|*false*
```

Comment:
The current behavior is to accept a TLS client certificate as sufficient for authentication, and treat the request as successfully authenticated by an anonymous user otherwise.  An explicit flag provides a way to make the client certificate a requirement even when no plugins are configured.